qa/rpc-tests/test_framework/mininode.py

```diff
@@ -54,6 +54,7 @@
 SAPLING_PROTO_VERSION = 170006
 BLOSSOM_PROTO_VERSION = 170008
 NU5_PROTO_VERSION = 170050
+# NU6_PROTO_VERSION = 170110
```

Comment:
Commenting this out now isn't going to make any difference, but it's correct if we add RPC tests that depend on NU6.